### DISCUSSION TITLE
Model erase function not working anymore

### DISCUSSION THREAD

**Discussion by zuhooo** [google/model-viewer#3786] [Q&A]
we found this morning that if we have 2 glb models, when switch between models, the previous model doesn't disappear, and the model viewer will show 2 overlapped models at the same time, this issue go on all the models the view by the model viewer. Any fix for this?

**Comment by adrianposteuca** [comment]
It is a known issue! 
You can try this workaround until it is fixed. Change the Modelviwer src source to v1.12.0

 src="https://cdn.jsdelivr.net/npm/@google/model-viewer@1.12.0/dist/model-viewer-umd.min.js"

**Reply by elalish** [reply]
Indeed, this is #3780, but the fact it slipped into unpkg has already been resolved. Sorry!
